src/utils/handle-style.ts

```diff
@@ -52,13 +52,7 @@ function injectStyle({
   const { insertAt } = ref
 
   if (document.getElementById(id)) {
-    // this should never happen because of `injected[type]`
-    if (process.env.NODE_ENV !== 'production') {
-      // eslint-disable-next-line no-console
-      console.warn(
-        `[react-tooltip] Element with id '${id}' already exists. Call \`removeStyle()\` first`,
-      )
-    }
+    // this could happen in cases the tooltip is imported by multiple js modules
```

Comment:
Is this even worth investigating further?

There seems to be no downside to just skipping injecting styles if they're already present.

Reply:
I do agree, the issue is inject the styles multiple times, but there's no issue if we only ignore it if it's already injected into the website